src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -113,21 +113,17 @@ export const NewProposal = ({ navigation, data, query }) => {
     setValue('tagList', filterData.toString());
   }, [tags]);
 
-  const uploadImage = async (imageData) => {
-    const imageUploadData = await uploadAttachment(imageData, 'image');
+  const uploadData = async (dataUri, dataType) => {
+    const { status, body } = await uploadAttachment(dataUri, dataType);
 
-    if (imageUploadData.status === 200) {
-      const { cached_attachment: cachedAttachment, filename: title } = JSON.parse(
-        imageUploadData.body
-      );
+    if (status === 200) {
+      const { cached_attachment: cachedAttachment, filename: title } = JSON.parse(body);
 
       return {
         title,
         cachedAttachment
       };
-    } else if (imageUploadData.status === 422) {
-      const errors = JSON.parse(imageUploadData.body).errors.toLowerCase().split(' ').join('-');
-
+    } else if (status === 422) {
       throw errors;
```

Comment:
which `errors` are thrown now? they are undefined, or am i wrong? i think we need to keep `const errors = JSON.parse(imageUploadData.body).errors` in ordern to see and probably log the server errors in a `catch (error)` block

Reply:
[096984c](https://github.com/ikuseiGmbH/smart-village-app-app/pull/536/commits/096984c64d992d9da3140016b63ccff108960dcf)